extension/pgtelemetry--1.3--1.4.sql

```diff
@@ -0,0 +1,68 @@
+create table @extschema@.long_running_query_rules
+(
+  priority                  int4,
+  application_name_ilike    text,
+  usename                   name,
+  state                     text,
+  alert_threshold           interval not null,
+  active_since              timestamp,
```

Comment:
Let's use timestamptz by default.

Reply:
we use timestamp every other place, but I agree

Reply:
and we should fix those as well when possible.

Reply:
What’s the benefit?

Reply:
none in out current setup

Reply:
The benefits is that the timezone information will be considered when saving such a timestamp (in other words, it will be converted to UTC). There are no drawbacks, as timestamptz and timestamp take the same amount of space.

Reply:
again, there is no difference in our setup, unless someone deliberately footguns, in which case nothing will help

Reply:
timestamptz is future-proof; we may change the decision to run our servers on UTC, we may have no choice in the future, or we may have clients actually setting their own timezone and writing data. None of that we control.